e2e/packages/contracts/mud.config.ts

```diff
@@ -47,5 +47,9 @@ export default mudConfig({
         player: "address",
       },
     },
+    StaticArray: {
+      keySchema: {},
+      valueSchema: "uint256[2]",
+    },
```

Comment:
do we have or want tests for this? or are e2e tests just being compiled/run enough?

Reply:
the table being deployed is enough, which happens in just e2e tests being run